packages/aws/src/lib/AWSRegions.ts

```diff
@@ -28,4 +28,5 @@ export enum AWS_REGIONS {
   SA_EAST_1 = 'sa-east-1',
   US_GOV_EAST_1 = 'us-gov-east-1',
   US_GOV_WEST_1 = 'us-gov-west-1',
+  UNKNOWN = 'unknown',
```

Comment:
Interested in why we're adding this -- is it just in case we don't know the region in the future so it doesn't break the CO2e conversion, like is probably happening with Azure?

I think that is a good reason to add it, but I also checked the `product_region` values in the CUR data, and it includes an option for `global`, so I'm thinking we add a `global` region as well, which will likely have the same value as `unknown` as it'll be the global average.